src/data.ts

```diff
@@ -44,7 +44,7 @@ export async function getSourceMap(
   url: string,
   source: string,
 ): Promise<RawSourceMap | undefined> {
-  const match = source.match(/\/\/# *sourceMappingURL=(.*)$/);
+  const match = source.match(/\/\/# *sourceMappingURL=(.*)/);
```

Comment:
```js
//# sourceURL=[module]
//# sourceMappingURL=//# sourceMappingURL=data:application/json;charset=utf-8;base64,...
//# sourceURL=webpack-internal:///
```
This was what appeared in my `.next/static/chunks/pages/*.js` folder

The problem is that `match` will be undefined because `//# sourceMappingURL` is not the last line in the file.

Reply:
Ah, nice catch! 

Does the footer literally appear like that in the file? The regex would then still yield `//# sourceMappingURL=data:application/json;charset=utf-8;base64,...` as the URL and that's not right either. 🤔

Reply:
Oh no.. I pasted the second line incorrectly. It should have read:
```js
//# sourceURL=[module]
//# sourceMappingURL=data:application/json;charset=utf-8;base64,...
//# sourceURL=webpack-internal:///
```
([regex101.com playground](https://regex101.com/r/JJxYCM/1))

Reply:
Great, thanks for confirming!